LBBT-Design-Patterns/ITaxStrategy.cs

```diff
@@ -1,14 +1,12 @@
-﻿using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
```

Comment:
Removed unused imports ("using" statements). It's automatic in resharper, don't know if it is in VS as standard.